apps/dialtone-documentation/docs/components/card.md

```diff
@@ -82,7 +83,21 @@ They should be easy to scan for relevant and actionable information. Elements, l
     <div>(footer slot)</div>
   </div>
 </div>
-```
+'
+vueCode='
+<dt-card class="d-w264">
+  <template #header>
+    (header slot)
+  </template>
+  <template #content>
+    (content slot)
+  </template>
+  <template #footer>
+    (footer slot)
+  </template>
+</dt-card>
```

Comment:
This coloring was fixed in the other PR right? probably just have to merge in staging.

![Screenshot 2024-03-07 at 1 09 56 PM](https://github.com/dialpad/dialtone/assets/64808812/a987fca9-6925-45e4-ac89-8fca3cbc6eef)

Reply:
Yes, looks good now:
<img width="461" alt="image" src="https://github.com/dialpad/dialtone/assets/89984179/bb9edd9b-c7c5-42f3-a55f-30dd74f086ea">